src/stateless/selectbox.vue

```diff
@@ -237,6 +255,16 @@ export default {
         setSearch: debounce(function (v) {
             this.searchTextValue = v
         }, 400),
+        loadAsyncOptions () {
+            if (typeof this.getOptions === 'function') {
```

Comment:
When is getOptions not a function?

Reply:
When it's not provided (could check if is null actually),

Reply:
Yes, let's check for null then